rules/autoconsent/inmobi.json

```diff
@@ -0,0 +1,29 @@
+{
+    "name": "inmobi",
+    "prehideSelectors": [".accept-cookie"],
+    "detectCmp": [
+        {
+            "exists": ".accept-cookie > .accept-cookie-inner"
```

Comment:
Class names are all very vague so combining two of them with `>` to make it more specific.